src/app/components/elements/PageTitle.tsx

```diff
@@ -72,14 +77,13 @@ export const PageTitle = ({
   }, [currentPageTitle]);
 
   return (
-    <h1
-      id="main-heading"
-      tabIndex={-1}
-      ref={headerRef}
-      className={`h-title h-secondary d-sm-flex ${className ? className : ""}`}
-    >
+    <h1 id="main-heading" tabIndex={-1} ref={headerRef} className={`h-title h-secondary d-sm-flex ${className ?? ""}`}>
       <div className="mr-auto" data-testid={"main-heading"}>
-        {formatPageTitle(currentPageTitle, disallowLaTeX)}
+        {editable ? (
+          <EditablePageTitle onEdit={onEdit!} currentPageTitle={currentPageTitle} />
```

Comment:
There is a way in which you can remove the need for `!`. Instead of passing `editable` as a prop of `PageTitle`, you can make it a local variable, and only pass `onEdit`. Then you do something like `const editable = onEdit !== undefined`. It makes it a bit more robust this way. The disadvantage is that the way you've currently implemented is quite clear to see when something is supposed to be editable. So it's up to you.

Reply:
Yeah that makes more sense, I didn't like the `!`, or the possibility that someone could put `editable` to true but not provide the necessary `onEdit` function.